examples/uart/main.cc

```diff
@@ -53,8 +53,9 @@ static osEvent uart_event;
 
 class CustomUART : public bsp::UART {
  public:
-  CustomUART(UART_HandleTypeDef *huart) : bsp::UART(huart) {}
+  using bsp::UART::UART;
```

Comment:
```CustomUART``` inherits ```UART``` and it should also inherit the constructor from ```UART```. Mind explaining why we are ```using``` the constructor of the parent class here?

Reply:
Nope, I don't think the constructor is inherited, at least not in C++11